.eslintignore

```diff
@@ -1,12 +1,10 @@
-bin/**
-build/**
-conf/**
-coverage/**
-docs/**
-scripts/**
-templates/**
-tests/bench/**
-tests/fixtures/**
-tests/performance/**
-tmp/**
-test.js
```

Comment:
I didn't know that trick. :)

Let me know if `scripts` is used for something else I am not aware of.